packages/cozy-mespapiers-lib/src/components/Actions/Items/importAuto.js

```diff
@@ -9,52 +10,66 @@ import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
 import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
 
 import Konnector from '../../../assets/icons/Konnectors.svg'
-import { getStoreWebLinkByKonnector } from '../../../helpers/getStoreWebLinkByKonnector'
+import { buildKonnectorsQueryBySlug } from '../../../helpers/queries'
 import withLocales from '../../../locales/withLocales'
 
-export const importAuto = ({ paperDefinition }) => {
+export const importAuto = ({ paperDefinition, importAutoOnclick }) => {
   return {
     name: 'importAuto',
     Component: withLocales(
       // eslint-disable-next-line react/display-name
       forwardRef((props, ref) => {
-        const client = useClient()
         const { t } = useI18n()
+        const navigate = useNavigate()
+        const { pathname } = useLocation()
+        const normalizePathname = pathname.split('/create')[0]
 
         const {
           konnectorCriteria: {
-            category: konnectorCategory,
-            name: konnectorName
+            name: konnectorName,
+            category: konnectorCategory
           } = {},
           label
         } = paperDefinition
 
-        const webLink = getStoreWebLinkByKonnector({
-          client,
-          konnectorName,
-          konnectorCategory,
-          redirectionPath: flag('harvest.inappconnectors.enabled')
-            ? `/paper/files/${label}/harvest/${konnectorName || ''}`
-            : undefined
-        })
+        const queryKonnector = buildKonnectorsQueryBySlug(konnectorName)
+        const { data: konnectors, ...konnectorsQueryLeft } = useQuery(
+          queryKonnector.definition,
+          queryKonnector.options
+        )
+        const isKonnectorsLoading = isQueryLoading(konnectorsQueryLeft)
+
+        const redirectPathSearchParam = flag('harvest.inappconnectors.enabled')
```

Comment:
I think we enabled this flag everywhere, so you can remove this condition. @doubleface you confirm that?

Reply:
I confirm that